apps/src/utils.js

```diff
@@ -1,5 +1,6 @@
-/* global define, $ */
-import Immutable from 'immutable';
+// @flow
```

Comment:
Yeah, by default you whitelist files with `@flow`. Once you're ready, you can enable this in `.flowconfig`:
```
[options]
all=true
```
and then just blacklist files that you don't want to be checked with `@noflow`.